sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -3590,6 +3873,35 @@ public partial class PredictiveValue : System.ClientModel.Primitives.IJsonModel<
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.PredictiveValue>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.PredictiveValue>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class Processor : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Processor>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Processor>
+    {
+        public Processor(Azure.ResourceManager.Monitor.Models.ProcessorType processorType, string name) { }
+        public Azure.ResourceManager.Monitor.Models.BatchProcessor Batch { get { throw null; } set { } }
+        public string Name { get { throw null; } set { } }
+        public Azure.ResourceManager.Monitor.Models.ProcessorType ProcessorType { get { throw null; } set { } }
+        Azure.ResourceManager.Monitor.Models.Processor System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Processor>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Processor>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Monitor.Models.Processor System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Processor>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Processor>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Processor>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ProcessorType : System.IEquatable<Azure.ResourceManager.Monitor.Models.ProcessorType>
```

Comment:
this name is quite generic, could we add some prefix to specify it?

Reply:
Renamed to `PipelineGroupProcessorType`